action.yaml

```diff
@@ -438,16 +438,22 @@ runs:
         cat .frogbot/frogbot-config.yml
       shell: bash  
     
-    - name: FrogBot
-      if: ${{ env.JF_URL }}    
-      uses: jfrog/frogbot@v2.8.8
+    - name: Frogbot
+      if: ${{ env.JF_URL }}
+      run: /opt/frogbot scan-pull-request
+      shell: bash
       env:
         JF_URL: ${{ env.JF_URL }}
         JF_ACCESS_TOKEN: ${{ env.JF_ACCESS_TOKEN }}
         JF_GIT_TOKEN: ${{ env.JF_GIT_TOKEN }}
+        JF_GIT_PROVIDER: 'github'
+        JF_PATH_EXCLUSIONS: ${{ env.JF_PATH_EXCLUSIONS }}
+        JF_GIT_OWNER: ${{ env.GIT_REPO_OWNER}}
```

Comment:
Where are these values coming? Can't they come from Github's context?

Reply:
These values will be passed by the user while calling the frogbot step of composite action as shown [here](https://github.com/pentaho/actions-common/blob/f7ca99a69427a1192991e76b956d9d6da318256e/.github/workflows/pr.yml#L150).